extensions/true-fantom/math.js

```diff
@@ -496,6 +543,17 @@
     exactly_cont_block({ A, B }) {
       return cast.toString(A).includes(cast.toString(B));
     }
+    clamp_block({ A, B, C }) {
+      return Math.min(Math.max(A, B), C);
```

Comment:
Consider making this block work both ways around; so if you put a smaller number in B than C it adjusts the equation.

Reply:
Thank you!